Dockerfile

```diff
@@ -0,0 +1,44 @@
+# build base image
+FROM maven:3-openjdk-11-slim as maven
+
+# copy pom.xml
+COPY ./pom.xml ./pom.xml
+
+# copy src files
+COPY ./src ./src
+
+# build
+RUN mvn package
+
+# final base image
+FROM openjdk:11-jre-slim
+
+# set deployment directory
+WORKDIR /EIDER
+
+# copy over the built artifact from the maven image
+COPY --from=maven /target/eider*.jar ./EIDER.jar
+
+#Settings
+ENV LOGGING_LEVEL_TAMU='INFO'
+ENV SERVER_PORT='9000'
+ENV DATABASE_URL='jdbc:postgresql://host.docker.internal:5432/eider'
+ENV DATABASE_DDL_AUTO='create-drop'
```

Comment:
We should also use Spring convention for setting properties through environment variables.